tests/smoke/obssmoke_test.py

```diff
@@ -0,0 +1,107 @@
+from typing import List
+from uuid import uuid4
+from time import time, sleep
+import pytest
+from smoke import conftest
+from aiohttp import ClientResponse
+from api_test_utils import env
+from api_test_utils import poll_until
+from api_test_utils.api_session_client import APISessionClient
+from api_test_utils.api_test_session_config import APITestSessionConfig
+
+
+def dict_path(raw, path: List[str]):
+    if not raw:
+        return raw
+
+    if not path:
+        return raw
+
+    res = raw.get(path[0])
+    if not res or len(path) == 1 or type(res) != dict:
+        return res
+
+    return dict_path(res, path[1:])
+
+def _base_valid_uri(nhs_number) -> str:
+    return f"FHIR/R4/Observation?patient.identifier=https://fhir.nhs.uk/Id/nhs-number|{nhs_number}"
+
+
+def _valid_uri(nhs_number, procedure_code) -> str:
+    return _base_valid_uri(nhs_number) + f"&procedure-code:below={procedure_code}"
+
+
+@pytest.fixture(scope='function')
+def authorised_headers(valid_access_token):
+    return {"Authorization": f"Bearer {valid_access_token}"}
+
+
+@pytest.mark.e2e
+@pytest.mark.smoketest
+@pytest.mark.asyncio
+async def test_check_observation_is_secured(api_client: APISessionClient):
+
+    async with api_client.get(_base_valid_uri("9999999990"), allow_retries=True) as resp:
+        assert resp.status == 401
+
+
+@pytest.mark.e2e
+@pytest.mark.smoketest
+@pytest.mark.asyncio
+async def test_observation_happy_path(test_app, api_client: APISessionClient, authorised_headers):
+
+    correlation_id = str(uuid4())
+    authorised_headers["X-Correlation-ID"] = correlation_id
+    authorised_headers["NHSD-User-Identity"] = conftest.nhs_login_id_token(test_app, nhs_number="9999999990")
+
+    async with api_client.get(
+        _base_valid_uri("9999999990"),
+        headers=authorised_headers,
+        allow_retries=True
+    ) as resp:
+        assert resp.status == 200
+        body = await resp.json()
+        assert "x-correlation-id" in resp.headers, resp.headers
+        assert resp.headers["x-correlation-id"] == correlation_id
+        assert body["resourceType"] == "Bundle", body
+        assert len(body["entry"]) == 1
+        assert body["entry"][0]["resource"]["resourceType"] == "Observation"
+        assert body["entry"][0]["resource"]["id"] == "FAKEFORTESTS1"
+        assert body["entry"][0]["resource"]["code"]["coding"][0]["code"] == "871555000"
```

Comment:
I would add:
assert body["entry"][0]["resource"]["code"]["coding"][0]["system"] == "http://snomed.info/sct"
assert body["entry"][0]["resource"]["code"]["coding"][0]["display"] == "{text proper for test type}"